.github/workflows/ci.yml

```diff
@@ -1,7 +1,14 @@
 name: CI/CD
 
 on:
-  push:
```

Comment:
Since you removed the `push` event that triggers workflow, after merging the PR workflow won't be triggered. 
One option is to use the `push` but specify the branch master, like this: 
```
push:
  branches:
    - master
```
Also the workflow won't be triggered when you push to the branch that isn't a PR yet (that worked until now).